projects/onto-search/src/lib/onto-search-facet/basic-facet/basic-facet.component.html

```diff
@@ -0,0 +1,17 @@
+<div>
```

Comment:
The name basic-facet is pretty confusing. Isn't this a multiselect/checkbox facet?

Reply:
renamed to checkbox-facet